mantle/platform/machine/do/cluster.go

```diff
@@ -41,7 +41,12 @@ func (dc *cluster) NewMachine(userdata *conf.UserData) (platform.Machine, error)
 		return nil, err
 	}
 
-	droplet, err := dc.flight.api.CreateDroplet(context.TODO(), dc.vmname(), dc.sshKeyID, conf.String())
+	isCustomImage, err := dc.flight.api.IsCustomImage(context.TODO())
```

Comment:
Was the intent to read `dc.flight.isCustomImage` here?

Reply:
I believe you are right :D, I missed that. I'll double check once I refactor I am not calling the method unnecessarily again. I can just do:
```
	droplet, err := dc.flight.api.CreateDroplet(context.TODO(), dc.vmname(), dc.sshKeyID, dc.flight.isCustomImage, conf.String())

``

Reply:
Well I don't think I need this when I refactor if I understood correctly. Update incoming.